_posts/2020-06-22-new-web-site.md

```diff
@@ -0,0 +1,37 @@
+---
+title:  "New ROOT Web Site !"
+layout: archive
+author: Olivier Couet
+---
+
+If you are reading this blog post it means you are surfing on the New ROOT Web Site !
+
+This new web site is a complete redesigned both on the the content point of view and on the
+the technology used to build it.
+
+Regarding the later we are now using the [Jekyll](https://jekyllrb.com) system. Jekyll is a
+simple, blog-aware, static site generator for personal, project, or organization sites.
+Written in Ruby by Tom Preston-Werner, GitHub's co-founder, it is distributed under the
```

Comment:
You seem to have copied this from Wikipedia. That's not nice - either reword or create a reference to signal where this is taken from.